components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -952,6 +985,42 @@ def getAnnotation(self, ns=None):
             return AnnotationWrapper._wrap(self._conn, rv[0].child, link=rv[0])
         return None
 
+    def getAnnotationCounts(self, ns=None):
```

Comment:
You're not using the ```ns``` parameter here. I don't think we need to support filtering by namespace yet, so this can simply be removed.